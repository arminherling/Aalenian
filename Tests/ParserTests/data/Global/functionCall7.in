define inner(x: bool) 
{ 
}

define outer()
{
    inner(false)
}
