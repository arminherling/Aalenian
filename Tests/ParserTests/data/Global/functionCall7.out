FunctionDefinitionStatement: {
    Name: inner
    Parameters(1): {
        Parameter: {
            Name: x
            Type: bool
        }
    }
    Block(0): {
    }
}
FunctionDefinitionStatement: {
    Name: outer
    Parameters(0): {
    }
    Block(1): {
        ExpressionStatement: {
            FunctionCall: {
                Name: inner
                Arguments(1): {
                    Argument: {
                        Bool: false
                    }
                }
            }
        }
    }
}
